src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,15 @@
+<hibernate-configuration>
```

Comment:
probably, that's the file from a previous hw)

Reply:
no, i was  testing dao in main method)

Reply:
gotcha)